spacer/config.py

```diff
@@ -121,6 +121,8 @@ def get_local_model_path():
 
 LOCAL_FIXTURE_DIR = os.path.join(os.path.dirname(
     os.path.abspath(__file__)), 'tests', 'fixtures')
+LOCAL_FEATURE_DIR = os.path.join(os.path.dirname(
```

Comment:
This is only used in the scripts, right? If so, let's remove it from here, and make it an input to the script CLI. The scripts are not part of the PIP package so they should be as stand-alone as possible.

Reply:
make sense, done